mypyc/ops.py

```diff
@@ -1200,13 +1207,15 @@ def __init__(self,
                  name: str,
                  class_name: Optional[str],
                  module_name: str,
+                 namespace: Optional[str],
```

Comment:
Could `class_name` and `namespace` be combined?

Reply:
Good call. Turns out we actually don't need the `namespace` field so I removed it.